HaikuPorter/ShellScriptlets.py

```diff
@@ -331,7 +331,8 @@ def getShellVariableSetters(shellVariables):
 
 	CMAKE=$portPackageLinksDir/cmd~cmake/bin/cmake
 	if [ ! -f $CMAKE ]; then
-		CMAKE=$(command -v cmake)
+		CMAKE=$(type -Pp cmake)
```

Comment:
```suggestion
		CMAKE=$(type -P cmake)
```

Reply:
I think it needs both -P (search only in path, not in functions and aliases) and -p (just return the file name, or nothing)

Reply:
perhaps even -fPp (f = suppress shell function lookup)

Reply:
It seems to work fine `-Pp`. The extra `p` is no big deal. Thanks for fix!